### PR TITLE
Address Jim's suggestions

### PR DESCRIPTION
**Pull request by SimonClifford**
Improve style and clarity, as per Jim's comments.

### CODE REVIEW

File: process_model/process_model.py [hunk 1]
```diff
@@ -1,72 +1,154 @@
-import click
-import re
+"""process_model outputs a Fortran module to interface to the given TF models
```
Comment (jdenholm) [inline process_model/process_model.py:1]: Uber pedantic: one-line docstrings should start with a capital letter and end with full stop.

File: process_model/process_model.py [hunk 2]
```diff
@@ -1,72 +1,154 @@
...
+    type=click.STRING,
+    help="signature def to use (default " "`serving_default`)",
+)
+@click.option(
+    "--indent",
+    default=4,
+    type=click.INT,
+    help="Indentation level of output Fortran code (default 4)",
+)
+def main(
```
Comment (jdenholm) [inline process_model/process_model.py:70]: Glorious!

Comment (SimonClifford) [reply process_model/process_model.py:70]: Not entirely sure `mypy` knows what to do with these but it doesn't complain.

File: process_model/process_model.py (hunk 1 above)
Comment (SimonClifford) [reply process_model/process_model.py:1]:
```suggestion
"""Code to output a Fortran module to interface to the given TF models.
```

**Review by jdenholm — APPROVED** [review]

**Pull request merged**